docs/ops.html

```diff
@@ -1654,298 +1653,248 @@ <h5><a id="kafka_streams_client_monitoring" href="#kafka_streams_client_monitori
         <td>The total number of tasks created.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
+        <td>kafka.streams:type=stream-thread-metrics,thread-id=([-.\w]+)</td>
       </tr>
       <tr>
         <td>task-closed-rate</td>
         <td>The average number of tasks closed per second.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
+        <td>kafka.streams:type=stream-thread-metrics,thread-id=([-.\w]+)</td>
       </tr>
       <tr>
         <td>task-closed-total</td>
         <td>The total number of tasks closed.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>skipped-records-rate</td>
-        <td>The average number of skipped records per second.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>skipped-records-total</td>
-        <td>The total number of skipped records.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
+        <td>kafka.streams:type=stream-thread-metrics,thread-id=([-.\w]+)</td>
       </tr>
  </tbody>
 </table>
 
 <h5><a id="kafka_streams_task_monitoring" href="#kafka_streams_task_monitoring">Task Metrics</a></h5>
-All the following metrics have a recording level of <code>debug</code>:
+All of the following metrics have a recording level of <code>debug</code>, except for metrics
+dropped-records-rate and dropped-records-total which have a recording level of <code>info</code>:
  <table class="data-table">
       <tbody>
       <tr>
         <th>Metric/Attribute name</th>
         <th>Description</th>
         <th>Mbean name</th>
       </tr>
+      <tr>
+        <td>process-latency-avg</td>
+        <td>The average execution time in ns, for processing.</td>
+        <td>kafka.streams:type=stream-task-metrics,thread-id=([-.\w]+),task-id=([-.\w]+)</td>
+      </tr>
+      <tr>
+        <td>process-latency-max</td>
+        <td>The maximum execution time in ns, for processing.</td>
+        <td>kafka.streams:type=stream-task-metrics,thread-id=([-.\w]+),task-id=([-.\w]+)</td>
+      </tr>
+      <tr>
+        <td>process-rate</td>
+        <td>The average number of processed records per second across all source processor nodes of this task.</td>
+        <td>kafka.streams:type=stream-task-metrics,thread-id=([-.\w]+),task-id=([-.\w]+)</td>
+       </tr>
+      <tr>
+        <td>process-total</td>
+        <td>The total number of processed records across all source processor nodes of this task.</td>
+        <td>kafka.streams:type=stream-task-metrics,thread-id=([-.\w]+),task-id=([-.\w]+)</td>
+      </tr>
       <tr>
         <td>commit-latency-avg</td>
-        <td>The average commit time in ns for this task. </td>
-        <td>kafka.streams:type=stream-task-metrics,client-id=([-.\w]+),task-id=([-.\w]+)</td>
+        <td>The average execution time in ns, for committing.</td>
```

Comment:
Just for my own education, why this one is set to ns instead of ms?

Reply:
It has been always that way. This was not part of KIP-444. I guess because since this metric is on task level, the values may be smaller than on thread level where the metrics measure ms.